ckan/cli/search_index.py

```diff
@@ -77,6 +77,43 @@ def clear(dataset_name: str):
         clear_all()
 
 
+def list_orphans(return_list: bool=False) -> list|None:
+    import ckan.logic as logic
+    search = logic.get_action('package_search')({},{
+                'q': '*:*',
+                'fl': 'id'})
+    indexed_package_ids = {r['id'] for r in search['results']}
+
+    import ckan.model as model
+    from sqlalchemy.sql import select
+    package_ids = {r[0] for r in select([model.package_table.c['id']]).execute()}
+
+    orphaned_package_ids = []
+    for indexed_package_id in indexed_package_ids:
+        if indexed_package_id not in package_ids:
+            orphaned_package_ids.append(indexed_package_id)
+
+    if return_list:
+        return orphaned_package_ids
+    from pprint import pprint
+    pprint(orphaned_package_ids)
```

Comment:
use `click.echo/secho`, like other CLI commands do

Reply:
Using `click.echo` now instead of `pprint`.